wg-security-audit/README.md

```diff
@@ -0,0 +1,28 @@
+<!---
+This is an autogenerated file!
+
+Please do not edit this file directly, but instead make changes to the
+sigs.yaml file in the project root.
+
+To understand how this file is generated, see https://git.k8s.io/community/generator/README.md
+--->
+# Security Audit Working Group
+
+Perform a security audit on k8s with a vendor and produce as artifacts a threat model and whitepaper outlining everything found during the audit.
+
+## Meetings
+* Regular WG Meeting: [Mondays at 13:00 PT (Pacific Time)](https://docs.google.com/document/d/1RbC4SBZBlKth7IjYv_NaEpnmLGwMJ0ElpUOmsG-bdRA/edit) (weekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=13:00&tz=PT%20%28Pacific%20Time%29).
+
+## Organizers
+
+* Jessie Frazelle (**[@jessfraz](https://github.com/jessfraz)**), Microsoft
+* Aaron Small (**[@aasmall](https://github.com/aasmall)**), Google
+* Joel Smith (**[@joelsmith](https://github.com/joelsmith)**), Red Hat
+
+## Contact
+* [Slack](https://kubernetes.slack.com/messages/)
+* [Mailing list]()
```

Comment:
@jessfraz the slack/ML links are missing.

Reply:
yeah we are going to use kubernetes-security-discuss@googlegroups.com and no slack for now unless people want it, just more to cleanup when we are done, do you think we need one?

Reply:
I think it's fine not to have a dedicated channel for the wg: there is a slack channel "#kubernetes-security" too. I was confused that the links were not working and I found out later that this page is generated from a template.
Thanks for the link - I just joined the mailing list.